client/clientservice/src/event_service.cpp

```diff
@@ -37,7 +37,12 @@ Status EventServiceImpl::StreamEventGroups(ServerContext* context,
   cc::SubscribeRequest request;
   request.event_group_id = proto_request->event_group_id();
 
-  auto callback = [stream](cc::EventGroup eg) {
+  auto callback = [this, stream](cc::SubscribeResult event) {
```

Comment:
Nit: since `event` here corresponds to event group, naming the `cc::SubscribeResult` type object as `event` is confusing to me. Can we rename it to - `subscribe_result` maybe?